Directory.Build.targets

```diff
@@ -8,6 +8,29 @@
     <BundledNETCorePlatformsPackageVersion>$(MicrosoftNETCorePlatformsPackageVersion)</BundledNETCorePlatformsPackageVersion>
   </PropertyGroup>
 
+  <!-- Source Build still uses a 3.0 SDK, which we need to retarget to 5.0 until the first 5.0 SDK is publicly released -->
```

Comment:
Can't we use a newer SDK in 'master' now?

Reply:
@wtgodbe could you file a follow up for this?

Reply:
https://github.com/aspnet/AspNetCore-Internal/issues/3016